master/internal/api_task.go

```diff
@@ -21,7 +21,7 @@ func (a *apiServer) GetTask(
 		return nil, err
 	}
 
-	t := &taskv1.Task{}
+	t := new(taskv1.Task)
```

Comment:
I recall there was some discussion about (not) using `new`? what was the outcome of it? in line 42 below it uses the different style.

Reply:
Not sure why I made this change initially. You're correct we should not be using `new`